sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/query.py

```diff
@@ -116,27 +107,25 @@ def main():
     parser = ArgumentParser(description=__doc__)
     parser.add_argument("--date", required=True)
     parser.add_argument("--project", default="moz-fx-data-shared-prod")
-    parser.add_argument("--dataset", default="addon_moderations_derived")
+    parser.add_argument("--dataset", default="analysis")
```

Comment:
This appears to have been accidentally committed from your test setup.
```suggestion
    parser.add_argument("--dataset", default="addon_moderations_derived")
```